src/test/java/seedu/address/testutil/LessonUtil.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.testutil;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CLASS_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DAY;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_END_TIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NUMBER_OF_OCCURRENCES;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_START_TIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_VENUE;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_ITEM;
+
+import seedu.address.logic.commands.AddLessonCommand;
+import seedu.address.model.lesson.Lesson;
+
+/**
+ * A utility class for Lesson.
+ */
+public class LessonUtil {
+
+    /**
+     * Returns an add lesson command string for adding the {@code lesson}.
+     */
+    public static String getAddLessonCommand(Lesson lesson) {
+        return AddLessonCommand.COMMAND_WORD + " "
+                + PREFIX_CLASS_INDEX + INDEX_FIRST_ITEM.getOneBased() + " "
+                + getLessonDetails(lesson);
+    }
```

Comment:
Perhaps this might allow more versatility for this method? Or alternatively it could be mentioned in the method description that it will always assume a class index of one.
```suggestion
    public static String getAddLessonCommand(Index moduleClassIndex, Lesson lesson) {
        return AddLessonCommand.COMMAND_WORD + " "
                + PREFIX_CLASS_INDEX + moduleClassIndex.getOneBased() + " "
                + getLessonDetails(lesson);
    }
```

Reply:
Added ```moduleClassIndex``` parameter into ```getAddLessonCommand```